### DISCUSSION TITLE
Serverless 与轻量级虚拟化 Firecracker · NSDI '20 - 面向信仰编程 · /papers-firecracker

### DISCUSSION THREAD

**Discussion by draveness** [draveness/blog-comments#642] [Announcements]
https://draveness.me/papers-firecracker/

**Comment by CNife** [comment]
厉害厉害，能超售二十倍

**Comment by csbo98** [comment]
好奇作者的博客是使用什么搭建的

**Comment by draveness** [comment]
> @csbo98
> 好奇作者的博客是使用什么搭建的

Hugo

**Comment by titanxxh** [comment]
性能差这么多 竟然还能超售

**Comment by draveness** [comment]
> @titanxxh
> 性能差这么多 竟然还能超售

因为 I/O 性能差，其他的都还好

**Comment by Aron-H-2019** [comment]
为什么IO性能差了这么多？有大佬分析过不

**Comment by changtimwu** [comment]
補充個好玩的, 有人試過用 i3.metal 上面開 4000 個 VM   https://www.youtube.com/watch?v=Eh0sTEN2azc
